tools/mtouch/Application.cs

```diff
@@ -1076,6 +1076,15 @@ void CopyAotData ()
 			}
 		}
 
+		public static void CopyMsymData (string src, string dest)
+		{
+			var dirInfo = new DirectoryInfo (src);
```

Comment:
You do not need to create an instance of `DirectoryInfo` if it's only to check for the existing of a directory. instead use `Directory.Exists` from https://msdn.microsoft.com/en-us/library/system.io.directory.exists(v=vs.110).aspx